enhancements/kube-apiserver/static-pods-for-single-node-via-ignition.md

```diff
@@ -0,0 +1,92 @@
+---
+title: static-pods-for-single-node-via-ignition
+authors:
+  - "@deads2k"
+reviewers:
+approvers:
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - https://github.com/sjenning/rhcos-kaio  
+replaces:
+superseded-by:
+---
+
+# Static Pods for Single Node Cluster via Ignition
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+1. Do we even have a use-case for spending the time building this thought experiment?
+2. Do we want to be able to run other operators on top of this?
+3. Do we want things like must-gather to function properly?
+
+## Summary
+
+A while back, Seth Jennings had a cool idea for trying to create a single node cluster using ignition.
+The cluster would be non-configurable after "creation", non-upgradable, non-HA.
+The cluster would only have etcd, kube-apiserver, kube-controller-manager, kube-scheduler.
+This is a description of how we could generate supportable static pods.
```

Comment:
What would be the outcome? Not really understanding the 'static pod' and 'supportability' in this context. The summary and motivation lack a justification and a why ...